docs/includes/generated_docs/schemas/beta.raw.tpp.md

```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## isaric
 
-A subset of the ISARIC data.
+A subset of the ISARIC data,
```

Comment:
Can we expand on what a subset means?  I assume this relates to time frame and columns, but might be something else too?

Reply:
I've added a mention that this is referring to a subset of the columns in TPP. (There are something like ~900 columns in TPP.)